genshin/models/genshin/constants.py

```diff
@@ -76,14 +76,14 @@ class DBChar(typing.NamedTuple):
     10000056: ("Sara", "Kujou Sara", "Electro", 4),
     10000057: ("Itto", "Arataki Itto", "Geo", 5),
     10000058: ("Yae", "Yae Miko", "Electro", 5),
-    10000059: None,
+    10000059: ("Heizo", "Shikanoin Heizou", "Anemo", 4),
     10000060: ("Yelan", "Hydro", 5),
     10000061: None,
     10000062: ("Aloy", "Cryo", 105),
     10000063: ("Shenhe", "Cryo", 5),
     10000064: ("Yunjin", "Yun Jin", "Geo", 4),
-    10000065: None,
-    10000066: ("Ayato", "Hydro", 5),
+    10000065: ("Shinobu", "Kuki Shinobu", "Electro", 4),
+    10000066: ("Ayato", "Kamisato Ayato", "Hydro", 5),
```

Comment:
Does the API return `Kamisato Ayato` by default? I would've sworn it was normally just `Ayato` for the name.

Reply:
Isn't the second field the character's english name? I see Ayato's english name on the Wiki is Kamisato Ayato.

Reply:
It's based on what exactly the API returns. Frankly this entire approach is not great and I'll probably need to find a better way to do this.